ayon_api/entity_hub.py

```diff
@@ -2971,3 +3003,156 @@ def _get_label_value(self):
         if not label or self._name == label:
             return None
         return label
+
+
+class VersionEntity(BaseEntity):
+    """Entity representing a version on AYON server.
+
+    Args:
+        entity_id (Union[str, None]): Id of the entity. New id is created if
+            not passed.
+        parent_id (Union[str, None]): Id of parent entity.
+        name (str): Name of entity.
+        attribs (Dict[str, Any]): Attribute values.
+        data (Dict[str, Any]): Entity data (custom data).
+        thumbnail_id (Union[str, None]): Id of entity's thumbnail.
+        active (bool): Is entity active.
+        entity_hub (EntityHub): Object of entity hub which created object of
+            the entity.
+        created (Optional[bool]): Entity is new. When 'None' is passed the
+            value is defined based on value of 'entity_id'.
+    """
+
+    entity_type = "version"
+    parent_entity_types = ["product"]
+
+    def __init__(
+        self,
+        *args,
+        tags=None,
```

Comment:
We should add `task_id`.

Reply:
done at https://github.com/ynput/ayon-python-api/pull/158/commits/50e75cdfe634ab773ecf31cd44b9986a0a8b18c0

is `comment` something passed as an `attrib`?